testing/backup-manager/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="backup-manager"
+pkgver="0.7.15"
+_pkgreal=Backup-Manager
+pkgrel=0
+pkgdesc="Backup Manager is a command line backup tool for GNU/Linux"
+url="https://github.com/sukria/Backup-Manager"
+arch="noarch"
+license="GPL-2.0-or-later"
+depends="gettext perl bash"
+makedepends="perl-file-slurp"
+options="!check" # no test suite or working check function
+source="$pkgname-$pkgver.tar.gz::https://github.com/sukria/Backup-Manager/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+prepare() {
+	sed -e "/^PERL5DIR/s/sitelib/vendorlib/" \
+	    -e "/sed/s:=\$(DESTDIR)/:=:" \
+	    -i Makefile
+}
+
+package() {
+	make PREFIX=/usr DESTDIR="$pkgdir" -C "$builddir" install_binary
```

Comment:
I don't think the `-C "$builddir"` is actually required here

Reply:
I'll make a new PR, you can close this one. Thanks;)

Reply:
Ah, you can just force push here

Reply:
I removed that branch;) 
Here's new PR #11908